pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -253,17 +253,15 @@ func newRuntimeVersion(version string) (*utilversion.Version, error) {
 func (m *kubeGenericRuntimeManager) getTypedVersion() (*runtimeapi.VersionResponse, error) {
 	typedVersion, err := m.runtimeService.Version(kubeRuntimeAPIVersion)
 	if err != nil {
-		klog.Errorf("Get remote runtime typed version failed: %v", err)
-		return nil, err
+		return nil, fmt.Errorf("get remote runtime typed version failed: %v", err)
```

Comment:
If anyone else is curious why capitalization changed here, looks like best practice is for logs to start with capital, but errors to not: https://github.com/kubernetes/kubernetes/issues/15863.

Reply:
@mattjmcnaughton thanks for the explanation